pubsub_nats_options.go

```diff
@@ -12,15 +12,35 @@
 package bahamut
 
 import (
-	"bytes"
 	"context"
 	"crypto/tls"
 	"fmt"
 	"time"
 
-	nats "github.com/nats-io/go-nats"
+	"github.com/nats-io/go-nats"
 )
 
+type requestMode int
+
+const (
+	requestModeNoReply requestMode = iota
```

Comment:
👀

Reply:
why are we not reusing the constants in publication.go?